### ISSUE TITLE
Docs on how to use the library?

### ISSUE THREAD

**Issue by piskvorky** [scaleway/postal-address#18]
Hi, the lib intro sounds great, but I couldn't find any doc examples on how to actually use it.

There's the API documentation at https://postal-address.readthedocs.io/en/develop/ (which is detailed but not helpful in figuring out what to do).

Is there a quick start example or something? How do I detect addresses in text using `postal-address`?

Apologies if I missed something obvious.

**Comment by gaskyk** [comment]
I agree with @piskvorky . I've just installed this package but unsure where I go from here. Thanks.

**Comment by srevenant** [comment]
yep

**Comment by ymz1124** [comment]
agree with @piskvorky

**Comment by EvaSDK** [comment]
Afaiu, current code limits itself to validating already split input data (line1, line2, postal_code, city_name) and formatting it. There is no address detection feature that I could see in the source code.

**Comment by KyleHaynes** [comment]
agree with @piskvorky as well

**Comment by dhk** [comment]
I'd be happy to write some docs/annotated examples if the owner could provide a use-case I can run with.

**Comment by thaichat04** [comment]
> Afaiu, current code limits itself to validating already split input data (line1, line2, postal_code, city_name) and formatting it. There is no address detection feature that I could see in the source code.

That leads us to misunderstand the function of this module: this help ready parsing/extract address from full text or not ?

**Comment by kotoroshinoto** [comment]
the docs seem to be unavailable now
